arangod/Cluster/ClusterInfo.cpp

```diff
@@ -437,8 +437,6 @@ bool ClusterInfo::doesDatabaseExist(DatabaseID const& databaseID, bool reload) {
   int tries = 0;
 
   if (reload || !_planProt.isValid || !_currentProt.isValid || !_DBServersProt.isValid) {
```

Comment:
I fear this does not make any sense any more. How do we ensure that `doesDatabaseExist` works in the early startup phase? Since the `loadPlan` and `loadCurrent` are gone, there is nothing here to prevent a wrong `false` response if the agency cache simply has no data yet... The `loadCurrentDBServers` does not help either, I think.